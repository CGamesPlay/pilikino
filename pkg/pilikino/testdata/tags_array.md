---
title: tags as a yaml array
tags: ['one', 'two']
---
