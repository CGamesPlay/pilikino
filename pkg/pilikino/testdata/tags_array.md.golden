{"filename":"","date":"0001-01-01T00:00:00Z","modified":"0001-01-01T00:00:00Z","title":"tags as a yaml array","tags":["one","two"],"links":null,"content":"","errors":null}